docs/sdks/server-sdks/java.md

```diff
@@ -69,6 +69,118 @@ The SDK will invoke the `logAssignment` function with an `event` object that con
 More details about logging and examples (with Segment, Rudderstack, mParticle, and Snowplow) can be found in the [event logging](/sdks/event-logging/) page.
 :::
 
+### Define a bandit assignment logger (contextual multi-armed bandit assignment only)
```

Comment:
I think it makes more sense to move the bandit section of the Java SDK to the bottom so that all the feature flag content is grouped and the bandit content is grouped separately